gym/wrappers/transform_reward.py

```diff
@@ -1,29 +1,41 @@
+"""Wrapper for transforming the reward."""
+from typing import Callable
+
+import gym
 from gym import RewardWrapper
 
 
 class TransformReward(RewardWrapper):
-    r"""Transform the reward via an arbitrary function.
-
-    Example::
+    """Transform the reward via an arbitrary function.
 
+    Example:
```

Comment:
Proper code highlighting on website
```suggestion
    Example::
```